SDK-267 [Python SDK] File storage README (#132)
incountry/sdk-python: README.md

        endpoint: Optional[str] = None,         # Optional. Defines API URL. Can also be set up through the INC_ENDPOINT environment variable
        encrypt: Optional[bool] = True,         # Optional. If False, encryption is not applied
        debug: Optional[bool] = False,          # Optional. If True enables the additional debug logging
        options: Optional[Dict[str, Any]] = {}, # Optional. It is used to fine-tune some configurations
        custom_encryption_configs: Optional[List[dict]] = None, # Optional. List of custom encryption configurations
        secret_key_accessor: Optional[SecretKeyAccessor] = None, # Instance of the SecretKeyAccessor class. It is used to fetch the encryption secret
    ):
        ...
```

---
**WARNING**

API Key authorization is being deprecated. The backward compatibility is preserved for the `api_key` parameter but you no longer can access API keys (neither old nor new) from your dashboard.

---

The `client_id`, `client_secret`,  and `environment_id` variables can be fetched from your dashboard on `Incountry` Portal.

The `endpoint` variable defines API URL and is used to override the default one.

> You can disable the encryption (not recommended). Set the `encrypt` property to `false` if you want to disable it. Avoid using it when using for production data.

Reviewer: ```suggestion
You can disable the encryption (not recommended). Set the `encrypt` property to `false` if you want to disable it. Avoid disabling it when working with production data.
```

**NOTE**

Attachments are currently available for InCountry dedicated instances only. Please check your subscription plan for details. This may require specifying your dedicated instance endpoint when configuring NodeJS SDK Storage.

---

InCountry Storage allows you to attach files to the previously created records. Attachments' meta information is available through the `attachments` field of `record` dictionary, returned by `read`, `find` and `find_one` methods.

```python
read_result = storage.read(country="us", record_key="<record_key>")

# read_result with attachments would be as follows
read_result = {
    "record": {
        "record_key": "user_1",
        ...,
        "attachments": [
            {
                "created_at": datetime,
                "updated_at": datetime,
                "download_link": "<download_url>",
                "file_id": "<file_id>",
                "filename": "<file_name>.<ext>",
                "hash": "<hash>",
                "mime_type": "<mime_type>",
                "size": 1,
            },
        ],
    },
}
```

### Adding attachments

---
Note:
---

The `add_attachment` method allows you to add or replace attachments.

> File data can be provided either as `BynaryIO` object or `string` with a path to the file in the file system.

Reviewer: ```suggestion
File data can be provided either as `BinaryIO` object or `string` with a path to the file in the file system.
```
